docs/modules/ROOT/pages/db_services/data_store.adoc

```diff
@@ -53,14 +53,14 @@ Similar analysis could be done for other controllers.
 ** The target is 1 million read/write operations per second.
 ** Estimated size of a middle-sized region is 500,000 node with 100 ports per node with consideration of multi-port container and VMs.
 Thus, the estimated total number of ports is ~50 millions.
-** The estimated number of port-related resources (e.g. security groups, ACL, routes, DNS records and subnets) is of the same magnitude.
+** The estimated number of port-related resources (e.g. security groups, ACL, routeEntities, DNS records and subnets) is of the same magnitude.
```

Comment:
I think we need to revert the changes in *.adoc, *.puml, *.proto and application.properties.

Global replacement is dangerous :-)

Reply:
It's my bad ~ I will fix them

Reply:
Yeah you are right ! it's my fault

Reply:
It is okay. It happened. Don't worry about it. 

Waiting for the latest CI results to merge the PR.